From c965cb8a004c6cc370b4bf297c61fe5ac8ab0583 Mon Sep 17 00:00:00 2001
From: Martijn Braam <martijn@brixit.nl>
Date: Wed, 6 Jan 2021 03:11:17 +0100
Subject: [PATCH] arm64: dts: sunxi: Add mmc aliases for the PineTab

The order for the mmc devices changed in the kernel without this change.

Signed-off-by: Martijn Braam <martijn@brixit.nl>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
index a87790df94b3..1cf3c3a9ad7f 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinetab.dts
@@ -18,6 +18,9 @@ / {
 	compatible = "pine64,pinetab", "allwinner,sun50i-a64";
 
 	aliases {
+		mmc0 = &mmc0;
+		mmc1 = &mmc1;
+		mmc2 = &mmc2;
 		serial0 = &uart0;
 		ethernet0 = &rtl8723cs;
 	};
-- 
2.29.2

